osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -373,11 +376,20 @@ public MenuItem[] ContextMenuItems
             {
                 List<MenuItem> items = new List<MenuItem>();
 
+                if (score.Mods.Length > 0 && modsContainer.Any(s => s.IsHovered))
+                    items.Add(new OsuMenuItem("Use these mods", MenuItemType.Highlighted, getMods));
+
                 if (score.ID != 0)
                     items.Add(new OsuMenuItem("Delete", MenuItemType.Destructive, () => dialogOverlay?.Push(new LocalScoreDeleteDialog(score))));
 
                 return items.ToArray();
             }
         }
+
+        private void getMods()
```

Comment:
Probably unnecessary to be extracted into a method, and the menu item should not be added in the first place if `songSelect` is null.